src/functions/create/mock.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What is this file used for? Should it be here?

Reply:
It would be useful should we have used some other means of testing.
With the `jest` testing currently implemented, this is not relevant anymore and was removed.